src/visualization/types/Table/view.tsx

```diff
@@ -27,6 +28,13 @@ import {
   DEFAULT_SORT_DIRECTION,
   KEYS_I_HATE,
 } from './constants'
+import {
+  Config,
+  DEFAULT_TABLE_COLORS,
+  HoverTimeProvider,
+  Plot,
+} from '@influxdata/giraffe'
+import {isFlagEnabled} from '../../../shared/utils/featureFlag'
```

Comment:
```suggestion
import {isFlagEnabled} from 'src/shared/utils/featureFlag'
```

webpack aliases `src` for us so we don't need to use relative paths. I think Sahas also had an issue with this with WebStorm